### ISSUE TITLE
Multiple elements with "banner" role do not have unique labels

### ISSUE THREAD

**Issue by alisonjoseph** [carbon-design-system/carbon-platform#1310]
**Bug🐛:** Running accessibility checker across a handful of pages reveals the following violation -  **Multiple elements with "banner" role do not have unique labels**
**Page:** https://next.carbondesignsystem.com/
**Browser:** Chrome - a11y plugin
**Suggestion:** To recreate, run scan using accessibility plugin.  It's pointing to the code <div aria-labelledby="consent_blackbar" role="banner" id="consent_blackbar">
<img width="1418" alt="Screen Shot 2022-09-19 at 4 05 12 PM" src="https://user-images.githubusercontent.com/23237857/191107342-c3ef1ae1-f69b-47e6-aa06-8f6311f951f2.png">

_Originally posted by @ljcarot in https://github.com/carbon-design-system/carbon-platform/discussions/1268#discussioncomment-3684324_

**Comment by andreancardona** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Development-624c3fc8b7972e001130fef7/issues/carbon-design-system/carbon-platform/1310?planning-poker) with Zenhub @alisonjoseph @francinelucca @jdharvey-ibm

**Comment by alisonjoseph** [comment]
Looks like this has to do with the cookie preferences banner that is added with the ibm-common js that we have no control over.

![Screen Shot 2022-09-23 at 8.35.59 AM.png](https://images.zenhubusercontent.com/5667369b638e718c995f3b3e/c86db1b1-363c-4b05-92e5-9114d5c00b80)
